src/networking.c

```diff
@@ -1349,13 +1349,43 @@ void acceptCommonHandler(connection *conn, struct ClientFlags flags, char *ip) {
         connClose(conn);
         return;
     }
+    in_addr_t ip_addr = 0;
+    if (sa != NULL && sa->ss_family == AF_INET) ip_addr = ((struct sockaddr_in *)sa)->sin_addr.s_addr;
```

Comment:
with #736 going into Valkey 8.0, I think we should support ipv6 trusted address as well. @zuiderkwast FYI